docs/io/configuration/components/index.rst

```diff
@@ -0,0 +1,39 @@
+.. _config-components:
+
+************************
+Configuration Components
+************************
+
+TARDIS uses the `YAML markup language <https://en.wikipedia.org/wiki/YAML>`_
+for its configuration files. There are several sections which allow different
+settings for the different aspects of the TARDIS calculation. An example
+configuration file (with a small subset of the options that can be specified) can be downloaded `here
+<https://raw.githubusercontent.com/tardis-sn/tardis/master/docs/tardis_example.yml>`_.
+
+.. warning::
+    One should note that currently floats in YAML need to be specified in a special format:
+    any pure floats need to have a +/- after the "e", e.g. 2e+5
```

Comment:
More correctly, any floats with _exponents_ need to have this format

Reply:
I will change that, this part was copied and pasted from the old documentation.